src/main/resources/db/migration/V27__add_merknader_sykmeldingsopplysninger.sql

```diff
@@ -0,0 +1 @@
+alter table sykmeldingsopplysninger ADD COLUMN merknader JSONB;
```

Comment:
alter table sykmeldingsopplysninger ADD COLUMN merknader JSONB null defualt null?

Reply:
Det blir kanskje det samme

Reply:
skal dobbeltsjekke :) Mener å huske at den tar veldig lang tid hvis man prøver å sette defaultverdier eksplisitt, og at det heller ikke er nødvendig.

Reply:
When a column is added with ADD COLUMN and a non-volatile DEFAULT is specified, the default is evaluated at the time of the statement and the result stored in the table's metadata. That value will be used for the column for all existing rows. If no DEFAULT is specified, NULL is used. In neither case is a rewrite of the table required.

Adding a column with a volatile DEFAULT or changing the type of an existing column will require the entire table and its indexes to be rewritten. As an exception, when changing the type of an existing column, if the USING clause does not change the column contents and the old type is either binary coercible to the new type or an unconstrained domain over the new type, a table rewrite is not needed; but any indexes on the affected columns must still be rebuilt. Table and/or index rebuilds may take a significant amount of time for a large table; and will temporarily require as much as double the disk space.

https://www.postgresql.org/docs/current/sql-altertable.html